### PR TITLE
Feature/docker io gramine base

### PR DESCRIPTION
**Pull request by jeremyjams**

### CODE REVIEW

File: cloud-computing/gramine/README.md [hunk 1]
```diff
@@ -1,90 +1,7 @@
...
-Depending on the error code, the issue is the following:
-
-| Error code |       Error       |                                                         Solution                                                          |
-|:----------:|:-----------------:|:-------------------------------------------------------------------------------------------------------------------------:|
-|    111     |  Can't reach SPS  |                                       Check SPS IP is correct in app configuration.                                       |
-|    9984    | Certificate error | Check both app & SPS share a valid certificate. Regenerate it if needed, providing SPS IP as `Common Name` when prompted. |
-
-
-#### Dataset and input files are not correctly read
-Check they are correctly added as `sgx.allowed_files` in `entrypoint.manifest`.
```
Comment (mcornaton) [inline cloud-computing/gramine/README.md:90]: We may keep these lines as this could help debug the app via Task Feedback.

File: Jenkinsfile [hunk 2]
```diff
@@ -1,33 +1,33 @@
 @Library('global-jenkins-library@2.3.1') _
 
 buildInfo = getBuildInfo()
+dockerIoVisibility = Registries.EXTERNAL_DOCKERIO_HOST
```
Comment (mcornaton) [inline Jenkinsfile:4]: Not sure why we declare a variable for an enum member but not for the other :thinking:

Comment (jeremyjams) [reply Jenkinsfile:4]: The first one is here to remove duplication :)

File: cloud-computing/gramine/README.md (hunk 1 above)
Comment (jeremyjams) [reply cloud-computing/gramine/README.md:90]: You are right, just added info in related story :+1: Thanks!

File: Jenkinsfile (hunk 2 above)
Comment (mcornaton) [reply Jenkinsfile:4]: I don't think having duplicates on enum members/constants is an issue. Do you think so?

Comment (jeremyjams) [reply Jenkinsfile:4]: It expresses here a same policy applies for different stages. Not sure why we are stuck here :thinking: :slightly_smiling_face:

Comment (mcornaton) [reply Jenkinsfile:4]: IMHO, using the `Registries.EXTERNAL_DOCKERIO_HOST` would express the same policy. I find it clearer to have a 'symmetrical' pattern: whether we use new variables for all visibilities, whether we don't use variables,
But you're right, we shouldn't be stuck there. That's very minor, just a personal preference. Let's keep moving!

**Review by mcornaton — APPROVED** [review]

**Review by jbern0rd — APPROVED** [review]
Thanks

**Pull request merged**